PolyEngine/Games/SandboxRendering/Src/RenderingSandbox.cpp

```diff
@@ -86,9 +98,20 @@ void RenderingSandbox::Init()
 	DeferredTaskSystem::AddComponentImmediate<Poly::MeshRenderingComponent>(world, ground, "Models/ground/ground.fbx", eResourceSource::GAME);
 	Poly::TransformComponent* groundTrans = world->GetComponent<Poly::TransformComponent>(ground);
 	//	groundTransform->SetLocalTranslation(Vector(SCALE * SIZE, 0.f, SCALE * SIZE));
-	groundTrans->SetLocalScale(SCALE);
+	// groundTrans->SetLocalScale(SCALE);
 
 	Engine->RegisterGameUpdatePhase(GameMainSystem::GameUpdate);
+
+	for (auto cmpTuple : world->IterateComponents<PostprocessSettingsComponent>())
```

Comment:
Looks hackish. Wouldn't it be better to just retrive the postprocess component from the object we're interested in?

Reply:
done, yes